src/PanelTraits/AutoSet.php

```diff
@@ -102,6 +152,9 @@ public function getFieldTypeFromDbColumnType($field)
             break;
 
             case 'text':
+                return 'textarea';
```

Comment:
This could become part of the next fall through.

PSR standard (2) or is it Symfony says that fall throughs WITHOUT comment are **evil** too btw.